CHANGELOG.md

```diff
@@ -2,31 +2,80 @@
 
 ## 0.8.0 (unreleased)
 
-The package name has changed from "crustls" to "rustls-ffi". If you are
-importing it as a library from other Rust code, you should import `rustls_ffi`.
-
-The header file is now named `rustls.h` instead of `crustls.h`, and the
-generated `.a` file is named `librustls_ffi.a` instead of `libcrustls.a`.
+The package name has changed to "rustls-ffi" (from "crustls").
+The header file (as installed by `make DESTDIR=/path/ install`)
+is now `rustls.h` and the library is `librustls.a`. The old library and header
+names are symlinked as part of the install process, to simplify upgrading to the
+new version.
+
+If you are importing this as a library from other Rust code, you should import `rustls_ffi`.
+
+## New
+ - rustls_{client,server}_config_builder_new_custom: start building a config,
```

Comment:
I'm worried if folks are searching the CHANGELOG for e.g. `rustls_server_config_builder_new_custom`, with cmd+f, they won't match this line because of the brackets and comma. (This is certainly something that I have done in the past).